### DISCUSSION TITLE
`An exception occurred during PROPFIND request on '/': password is mandatory in simple bind`

### DISCUSSION THREAD

**Discussion by poperigby** [Kozea/Radicale#1649] [Q&A server] [answered]
I'm running Radicale with the following configuration, and I'm getting this error when starting it:

```
An exception occurred during PROPFIND request on '/': password is mandatory in simple bind
```

```ini
[auth]
lc_username=true
ldap_base=dc=haddock,dc=cc
ldap_filter=(&(objectClass=person)(uid={0}))
ldap_reader_dn=uid=radicale,ou=people,dc=haddock,dc=cc
ldap_uri=ldap://localhost:3890
type=ldap
```

I also have an environment file with this:

```bash
RADICALE_OPTIONS="--auth-ldap-secret=REDACTED"
```

**Comment by pbiering** [comment]
@poperigby  - which version are you using?

@petervarkoly  - can you please investigate?

**Reply by poperigby** [reply]
I'm using 3.3.1.

**Comment by pbiering** [comment, marked as answer]
Cannot reproduce, are you sure that RADICALE_OPTIONS is used on startup?

See here how to use https://github.com/Kozea/Radicale/pull/1577 (end of thread)

If you use latest changes ldap config details will be shown on startup: https://github.com/Kozea/Radicale/pull/1651

**Reply by poperigby** [reply]
I just had to use auth.ldap_secret_file in my main config and that worked! Thanks!
